RocketCastUITests/PodcastViewUITest.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  PodcastViewUITest.swift
+//  RocketCast
+//
+//  Created by James Park on 2016-10-30.
+//  Copyright © 2016 UBCLaunchPad. All rights reserved.
+//
+
+import XCTest
+class PodcastViewUITest: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+        
+        let app = XCUIApplication()
+        app.launchArguments = ["MY_UI_TEST_MODE"]
+        app.launch()
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+       
+        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testForDuplicatePodcastsAndEpisodes() {
```

Comment:
This method is testing 2 things at once; podcasts and episodes. It might be better to split them up, what do you think?